.github/workflows/license.yaml

```diff
@@ -0,0 +1,33 @@
+################################################################################
+# This file is AUTOGENERATED with <https://github.com/sapcc/go-makefile-maker> #
+# Edit Makefile.maker.yaml instead.                                            #
+################################################################################
+
+name: License
+"on":
+  pull_request:
+    branches:
+    - '*'
+    paths-ignore:
+    - README.md
+  push:
+    branches:
+    - main
+    paths-ignore:
+    - README.md
+jobs:
+  addlicense:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: "1.16"
+    - name: Check if source code files have license header
+      run: |
+        GO111MODULE=off go get -u github.com/google/addlicense
+        find * -type d -prune -not -name vendor -o -name '*.go' -exec addlicense -check -- {} +
```

Comment:
This does not do what I expect. There are probably some parentheses missing. For example in sapcc/castellum, I get
```
$ find * -type d -prune -not -name vendor -o -name '*.go' -exec echo {} +
main.go
```
All Go files in subdirectories are skipped because of `-type d -prune` in front. Have another look at the original `find` invocation that I suggested, that one worked correctly as far as I remember.